mesh/pmesh.hpp

```diff
@@ -336,8 +344,11 @@ class ParMesh : public Mesh
    explicit ParMesh(const ParMesh &pmesh, bool copy_nodes = true);
 
    /// Read a parallel mesh, each MPI rank from its own file/stream.
-   /** The @a refine parameter is passed to the method Mesh::Finalize(). */
-   ParMesh(MPI_Comm comm, std::istream &input, bool refine = true);
+   /** The @a generate_edges parameter is passed to Mesh::Loader. The @a refine
+       and @a fix_orientation parameters are passed to the method
+       Mesh::Finalize(). */
+   ParMesh(MPI_Comm comm, std::istream &input, bool refine = true,
+           int generate_edges = 0, bool fix_orientation = true);
```

Comment:
For backward compatibility, we should use default value of `1` for `generate_edges` -- this is what's currently used by the constructor.

Reply:
Fixed in [0fa1275](https://github.com/mfem/mfem/pull/3918/commits/0fa1275ccf953f41fe10469fa828be33f89a2c8b), and added a comment to the docstring warning about the differences with `Mesh`.